<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>Widget</class>
 <widget class="QWidget" name="Widget">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>400</width>
    <height>300</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>Widget</string>
  </property>
  <layout class="QGridLayout" name="gridLayout">
   <item row="1" column="0">
    <widget class="QPushButton" name="btn01">
     <property name="styleSheet">
      <string notr="true">
    border-style: outset;
    border-width: 2px;
    border-radius: 10px;
    border-color: beige;
    font: bold 14px;
    min-width: 10em;
    padding: 6px;</string>
     </property>
     <property name="text">
      <string>Page 1</string>
     </property>
    </widget>
   </item>
   <item row="1" column="1">
    <widget class="QPushButton" name="btn02">
     <property name="styleSheet">
      <string notr="true">
    border-style: outset;
    border-width: 2px;
    border-radius: 10px;
    border-color: beige;
    font: bold 14px;
    min-width: 10em;
    padding: 6px;</string>
     </property>
     <property name="text">
      <string>Page 2</string>
     </property>
    </widget>
   </item>
   <item row="0" column="0" colspan="2">
    <widget class="QStackedWidget" name="stackedWidget">
     <widget class="ClassA" name="page1"/>
     <widget class="ClassB" name="page2"/>
    </widget>
   </item>
  </layout>
 </widget>
 <layoutdefault spacing="6" margin="11"/>
 <customwidgets>
  <customwidget>
   <class>ClassA</class>
   <extends>QWidget</extends>
   <header>ClassA.h</header>
   <container>1</container>
  </customwidget>
  <customwidget>
   <class>ClassB</class>
   <extends>QWidget</extends>
   <header>ClassB.h</header>
   <container>1</container>
  </customwidget>
 </customwidgets>
 <resources/>
 <connections/>
</ui>
